docs/command_line_reference.rst

```diff
@@ -41,68 +44,99 @@ command above).
 
 It further supports the following additional arguments:
 
-:``--checkout``: deletes the local Git repository and (re-)clones it.
-:``--compile``: (re-)compiles the build.
-:``--configure``: (re-)configures the build.
-:``--delete-source``: deletes the source directory when purging.
-:``--install``: (re-)installs the build.
-:``--purge``: deletes all related build files.
-:``--recheckout``: deletes the local build Git repository, reclones, regenerates, reconfigures, recompiles
-    and reinstalls it.
-:``--recompile``: recompiles and reinstall the build.
-:``--reconfigure``: reconfigures, recompiles and reinstalls the build.
-:``--regenerate``: (re-)regenerates the build.
-:``--reinstall``: reinstalls the build.
-:``--reregenerate``: regenerates, reconfigures, recompiles and reinstalls the build.
+:--checkout:      Deletes the local Git repository and clones it.
+:--compile:       Compiles the build.
+:--configure:     Configures the build.
+:--delete-source: Deletes the source directory when purging.
+:--install:       Installs the build files.
+:--purge:         Deletes all related build files.
+:--recheckout:    Deletes the cloned git repository, re-clones, regenerates,
+                  reconfigures, re-compiles, and reinstalls it.
+:--recompile:     Re-compiles and reinstall the build.
+:--reconfigure:   Re-configures the build.
+:--regenerate:    Re-generates the build.
+:--reinstall:     Re-installs the build.
```

Comment:
We should be consistent concerning the prefix "re". According to the Oxford Dictionary, it appears that writing all in one word is the correct scheme. Of course, there are some words, which the dictionary doesn't contain.

Reply:
Done.